src/Scrolling.md

```diff
@@ -62,3 +62,6 @@ For the window to be displayed on a scanline, the following conditions must be m
 
 If the WY condition has already been triggered and at the start of a row the window enable bit was set,
 then resetting that bit before the WX condition gets triggered on that row yields a nice window glitch pixel where the window would have been activated.
+
+The way the Window selects which line of its tilemap to render may be surprising: the Y position is selected by an internal counter, which is reset to 0 during VBlank and **only** incremented when the Window starts being rendered on a given scanline.
+In particular, this means that hiding the Window mid-frame in any way (via either `WX` or `LCDC`, usually to display a status bar at the top *and* bottom of the screen) will also inhibit incrementing that Y-position counter.
```

Comment:
I think this should also reference [this tip in the Tile Maps page](https://github.com/gbdev/pandocs/blob/master/src/Tile_Maps.md?plain=1#L107) or even move it / merge them.

Reply:
Agree, this is now a "repetition", we could actually merge it there and link it from here or just keep it here and link it from there.. maybe let's open an issue about this so we don't block this one?

Reply:
I opened https://github.com/gbdev/pandocs/issues/592